### PR TITLE
ascii-ipa:1.0.0

### PR DESCRIPTION
**Pull request by imatpot**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: ascii-ipa allows you to easily convert different ASCII representations of the International Phonetic Alphabet (IPA) to and from the IPA. It also offers some minor utilities to make phonetic transcriptions easier to use overall.

### CODE REVIEW

File: packages/preview/ascii-ipa-1.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+| `value`         | `string`  | positional         |         | Main input to the function. Usually the transcription in the corresponsing ASCII-based notation.                                 |
+| `reverse`       | `boolean` | named              | `false` | Reverses the conversion. Pass Unicode IPA into `value` to get the corresponsing ASCII-based notation back.                       |
+| `override-font` | `boolean` | named              | `false` | Overrides the active font and forces rendering in Linux Libertine. Use this if your font has lackluster support for Unicode IPA. |
+
+#### Examples
+
+All examples use the Russian word ⟨привет⟩ [prʲɪvʲet] for the conversion.
+
+```ts
+import "@preview/ascii-ipa:1.0.0": *
```
Comment (laurmaedje) [inline packages/preview/ascii-ipa-1.0.0/README.md:35]:
```suggestion
#import "@preview/ascii-ipa:1.0.0": *
```

**Review by laurmaedje — COMMENTED** [review]
Found one small mistake, looks good otherwise!

File: packages/preview/ascii-ipa-1.0.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[package]
+name = "ascii-ipa"
+version = "1.0.0"
+entrypoint = "lib.typ"
+authors = ["imatpot"]
+license = "MIT"
+description = "Converter for ASCII representations for the International Phonetic Alphabet (IPA)"
```
Comment (laurmaedje) [inline packages/preview/ascii-ipa-1.0.0/typst.toml:7]: one more thing: should it be "for the International Phonetic Alphabet" or maybe rather "of the International Phonetic Alphabet" here? I'm not sure.

Comment (imatpot) [reply packages/preview/ascii-ipa-1.0.0/typst.toml:7]: I guess "of" makes slightly more sense

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**